hibernate-reactive-core/src/main/java/org/hibernate/reactive/session/impl/ReactiveSessionImpl.java

```diff
@@ -1316,12 +1312,12 @@ public boolean isOrderReturnEnabled() {
 		if ( ReactiveSession.class.isAssignableFrom( clazz ) ) {
 			return clazz.cast(this);
 		}
-		if ( Stage.Session.class.isAssignableFrom( clazz ) ) {
-			return clazz.cast( new StageSessionImpl( this ) );
-		}
-		if ( Mutiny.Session.class.isAssignableFrom( clazz ) ) {
-			return clazz.cast( new MutinySessionImpl( this ) );
-		}
+//		if ( Stage.Session.class.isAssignableFrom( clazz ) ) {
+//			return clazz.cast( new StageSessionImpl( this ) );
+//		}
+//		if ( Mutiny.Session.class.isAssignableFrom( clazz ) ) {
+//			return clazz.cast( new MutinySessionImpl( this ) );
+//		}
```

Comment:
was this a leftover of some experiment?

If not, would be nice to add a comment about the reason.

Reply:
I don't think we need it. I think we can just delete that stuff.

Reply:
sure. I'd delete it then - the comments made me wonder if there was some unfinished business

Reply:
done